platform-cloud/docs/studios/overview.md

```diff
@@ -5,7 +5,7 @@ date: "6 February 2025"
 tags: [studios]
 ---
 
-Studios is a unified platform where you can host a combination of container images and compute environments for interactive analysis using your preferred tools, like JupyterLab and RStudio Notebooks, Visual Studio Code IDEs, or Xpra remote desktops. Each Studio session is an individual interactive environment that encapsulates the live environment for dynamic data analysis.
+Studios is a unified platform where you can host a combination of container images and compute environments for interactive analysis using your preferred tools, like JupyterLab notebooks, an R-IDE, Visual Studio Code IDEs, or Xpra remote desktops. Each Studio session is an individual interactive environment that encapsulates the live environment for dynamic data analysis.
```

Comment:
My understanding was that JupyterLab **N**otebooks is a named product. Similarly, an R IDE could be VS Code, so maybe R IDE should be plural here?

Reply:
In this case, "R-IDE" is a replacement for RStudio so it was a direct replacement and is why VS Code is mentioned separately because previously it was "RStudio". It's a fair point though, because it's now a "generic" vs. a specific product.

Reply:
I think we are overcomplicating this. It's a dedicated R IDE. Sure, other IDE's support R, but this is specifically just for an R IDE and is an equivalent of RStudio. It needs to be obvious what this is.